web/src/features/charts/graphUtils.ts

```diff
@@ -71,26 +71,38 @@ export const getGenerationTypeKey = (name: string): GenerationType | undefined =
   return undefined;
 };
 
-export function getTotalElectricity(
-  zoneData: ZoneDetail,
-  displayByEmissions: boolean,
-  mixMode: Mode
-) {
-  const includeImports = mixMode === Mode.CONSUMPTION;
-  let productionValue: number;
-
-  if (displayByEmissions) {
-    productionValue = zoneData.totalCo2Production + zoneData.totalCo2Discharge;
-    productionValue += includeImports ? zoneData.totalCo2Import : 0;
-  } else {
-    productionValue = zoneData.totalProduction + zoneData.totalDischarge;
-    productionValue += includeImports ? zoneData.totalImport : 0;
+export function getTotalElectricity(zoneData: ZoneDetail, mixMode: Mode) {
```

Comment:
Nice that you're splitting this function up, it was quite confusing that it did both! :)

Reply:
Yeah I started writing getTotalEmissionOrElectricity when I figured this was probably time to split it up